server/src/main/java/org/elasticsearch/index/mapper/CompletionFieldMapper.java

```diff
@@ -142,7 +142,7 @@ private static CompletionFieldMapper toType(FieldMapper in) {
             m -> toType(m).maxInputLength, Defaults.DEFAULT_MAX_INPUT_LENGTH)
             .addDeprecatedName("max_input_len")
             .setValidator(Builder::validateInputLength)
-            .alwaysSerialize();
+            .setSerializerCheck((id, ic, v) -> true);
```

Comment:
I wonder if `alwaysSerialize` should a shortcut to this new thing - it is easier for me to read the intent of `alwaysSerialize` than this lambda.

Reply:
I added `alwaysSerialize` and `neverSerialize` as shortcuts.